clang/lib/CodeGen/CGOpenMPRuntime.cpp

```diff
@@ -9576,6 +9576,20 @@ static void genMapInfo(const OMPExecutableDirective &D, CodeGenFunction &CGF,
                         MappedVarSet, CombinedInfo);
   genMapInfo(MEHandler, CGF, CombinedInfo, OMPBuilder, MappedVarSet);
 }
+
+static void emitNumTeamsForBareTargetDirective(
+    CodeGenFunction &CGF, const OMPExecutableDirective &D,
+    llvm::SmallVectorImpl<llvm::Value *> &NumTeams) {
```

Comment:
```suggestion
    llvm::ArrayRef<llvm::Value *> NumTeams) {
```

Reply:
Nah, I need to call `push_back` or something similar in the function.

Reply:
Ah, missed it